src/utils/config.ts

```diff
@@ -186,6 +199,7 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   TOKEN_EXPIRATION: fromNullableE(-1)(process.env.TOKEN_EXPIRATION)
     .chain(_ => IntegerFromString.decode(_).mapLeft(() => -1))
     .fold(() => 3600, identity),
+  USER_REGISTRY_URL: fromNullable(process.env.USER_REGISTRY_URL).getOrElse(""),
```

Comment:
Since `USER_REGISTRY_URL` is a `NonEmptyString` we don't need this.

Reply:
Fix with 14fd219